changelog_5c1ce29804286304a71d9ba4.md

```diff
@@ -2,6 +2,18 @@ Thank you for using Semaphore!
 We continuously deploy changes that improve the product for you.
 This page is updated on a weekly basis.
 
+### Week of September 30, 2019
+  
+- [The latest release of the Semaphore Agent](https://github.com/semaphoreci/agent/pull/72)
+  checks if Bash is available before starting jobs in a custom Docker container. Previously, 
+  without this check, the first step in a job that exports environment variables would fail
+  without a real indication of the root problem.
+- From now on, outdated version of the CLI are rejected from accessing your organization.
```

Comment:
outdated versions of CLI will be rejected

Reply:
Updated it.

---

Can you share the reason for this one, why future tense instead of simple present?
Until now, I always used simple present to represent actions that happen regularly or unceasingly.